src/app/components/views/login-view/login-view.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { CookieService } from 'ngx-cookie-service';
+
+@Component({
+  selector: 'app-login-view',
+  templateUrl: './login-view.component.html',
+  styleUrls: ['./login-view.component.scss']
+})
+export class LoginViewComponent implements OnInit {
+
+  public uidForm!: FormGroup; // The form for logging in to ABC
```

Comment:
Why `!`?

Reply:
I don't know what we decided but it's because he isn't instantiating it here or in the constructor. It's a angular.json setting I think